File: packages/sdks/javascript/src/services/PipelineService.ts
import { PipelinesActionsApiService } from "../api/generated/apis/pipelines-actions-api";
import { PipelinesHooksApiService } from "../api/generated/apis/pipelines-hooks-api";
import { PipelinesStepsApiService } from "../api/generated/apis/pipelines-steps-api";
import { PipelinesApiService } from "../api/generated/apis/pipelines-api";

export class PipelineService extends PipelinesApiService {
    public actions = new PipelinesActionsApiService();
    public hooks = new PipelinesHooksApiService();
    public steps = new PipelinesStepsApiService();
}

File: packages/sdks/python/src/devopness/services/pipeline_service.py
"""
Devopness API Python SDK - Painless essential DevOps to everyone
"""

from ..generated.api.pipelines_api import (
    PipelinesApiService,
    PipelinesApiServiceAsync,
)
from ..generated.api.pipelines_steps_api import (
    PipelinesStepsApiService,
    PipelinesStepsApiServiceAsync,
)

__all__ = ["PipelineService", "PipelineServiceAsync"]


class PipelineService(
    PipelinesApiService,
    PipelinesStepsApiService,
):
    """Service for pipelines in the Devopness API."""


class PipelineServiceAsync(
    PipelinesApiServiceAsync,
    PipelinesStepsApiServiceAsync,
):
    """Async service for pipelines in the Devopness API."""
